rasa_nlu/persistor.py

```diff
@@ -23,7 +23,7 @@ def get_persistor(config):
         raise KeyError("No persistent storage specified. Supported values are {}".format(", ".join(['aws', 'gcs'])))
 
     if config['storage'] == 'aws':
-        return AWSPersistor(config['path'], config['aws_region'], config['bucket_name'])
+        return AWSPersistor(config['path'], config['aws_region'], config['bucket_name'], config.as_dict().get('aws_endpoint_url'))
```

Comment:
The configuration value needs to be added to `config_defaults.json` as well as into the dict `DEFAULT_CONFIG` in `config.py`. (probably with the default value `None`). 

After that is done, the access here should be `config['aws_endpoint_url']`